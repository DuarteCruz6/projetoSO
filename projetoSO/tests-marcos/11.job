WRITE [(d7,10)(i7,74)(k1,16)(z4,37)]
WRITE [(i1,56)]
WRITE [(b7,93)(h5,23)(k9,52)]
READ [k1]
WRITE [(r0,95)(s4,19)]
WRITE [(q1,96)(g9,85)(u5,71)(e6,59)(z2,6)]
DELETE [h5,b7,z2,g9,d7]
WRITE [(a0,63)(h1,26)(f1,40)(h5,49)]
WRITE [(w0,57)(f0,36)(p2,61)(i3,80)]
WRITE [(x2,57)(f7,44)(m5,4)(c3,7)]
READ [f7]
WRITE [(r5,77)(p5,50)(e4,21)]
WAIT 3000
READ [x2]
READ [k9]
WRITE [(e5,67)(i2,72)(t9,7)(l1,74)]
WRITE [(y7,6)]
DELETE [h5,r5,h1]
DELETE [p5,e5,f1]
WRITE [(o8,59)(z2,67)]
DELETE [y7]
WAIT 3000
WRITE [(c1,95)(c1,59)(a5,10)(s7,39)(c7,57)]
DELETE [x2,i7,a0,z4]
DELETE [e4]
WRITE [(d4,91)(g6,4)(e9,11)(q2,10)(e5,4)]
WRITE [(z8,89)]
WRITE [(z9,91)(m3,71)(w7,12)(s0,86)]
READ [o8]
READ [g6]
WAIT 2000
WRITE [(k8,35)(d6,96)]
DELETE [c3,u5,f0,d6,i2]
WRITE [(q8,51)]
WRITE [(j9,56)(v4,74)]
SHOW
READ [i1]
READ [q8]
SHOW
READ [g6]
WRITE [(w3,98)(l6,38)(c2,4)(n6,3)]
WRITE [(m9,56)(g2,84)(u0,61)]
WRITE [(x0,31)(c5,3)(x7,24)]
WRITE [(q5,72)(o2,88)(i4,0)(e8,98)]
WRITE [(m7,65)(l3,19)(t3,39)(h5,56)(z4,76)]
BACKUP
READ [i1]
WRITE [(o1,86)(y5,74)(r1,16)(d5,26)(s3,66)]
WRITE [(d2,66)(g8,25)(n8,97)]
WRITE [(g3,95)]
WRITE [(p7,36)(l8,78)]
WRITE [(y6,48)(f4,22)(w6,35)]
READ [c2]
DELETE [e8,p7,p2,c1,s0]
READ [k9]
WRITE [(f1,100)(b0,94)(e3,23)]
BACKUP
DELETE [e3,e6]
WRITE [(h1,64)(i5,2)]
WRITE [(q4,14)(d9,97)]
READ [g3]
WRITE [(k3,90)(u8,9)]
DELETE [m5,d4,i5,x0]
WRITE [(o7,69)(w4,51)(r7,42)(m4,37)]
WRITE [(p0,73)]
WRITE [(i9,99)]
WRITE [(t1,52)(i0,91)(y3,37)(y3,10)(s6,72)]
WRITE [(h8,26)(p2,63)(a2,62)(s9,57)(y7,71)]
DELETE [j9,g6,e9,i9,o2]
SHOW
READ [m9]
READ [m3]
READ [q1]
DELETE [s3,d5,k3,h5,q5]
WRITE [(o9,60)(p6,88)(c6,92)]
WRITE [(w8,68)(h6,34)(s3,91)]
WRITE [(j6,9)(q6,86)]
WRITE [(o3,58)(o5,65)(h9,90)(f9,41)]
READ [l3]
WRITE [(j0,82)(d7,10)(f2,36)(d1,81)(r5,61)]
WRITE [(s8,76)(a0,2)(c4,79)(b1,66)(u9,96)]
WRITE [(j2,58)(g0,29)(m5,93)(b3,90)]
WRITE [(p1,9)(j8,89)(u4,66)(l4,68)]
WRITE [(m8,14)(w5,67)(g1,48)]
READ [f1]
WRITE [(d3,71)(v1,71)(g6,74)(p5,41)(q9,20)]
