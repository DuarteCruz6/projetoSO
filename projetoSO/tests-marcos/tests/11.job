WRITE [(a5,66)(l2,77)(d7,15)]
BACKUP
WRITE [(n2,59)(n1,61)(j8,1)(b2,99)]
WRITE [(d9,44)(b8,49)(k7,66)]
WRITE [(t2,57)(p7,50)]
WRITE [(j4,54)(c8,75)(f5,32)(j6,16)]
READ [j4]
WRITE [(h1,93)(q7,32)]
WRITE [(d2,46)(w1,88)]
DELETE [n1,j8]
WRITE [(b0,2)(y9,2)]
WRITE [(i9,2)]
READ [n2]
SHOW
WRITE [(o3,16)(u0,82)]
WRITE [(d6,23)(n9,20)]
WRITE [(w6,89)(s8,28)]
WAIT 2000
READ [b2]
WRITE [(o8,1)(r0,34)(t3,63)(i0,64)]
READ [w6]
WRITE [(m9,6)(a2,52)(a0,94)(z1,42)]
WRITE [(p3,19)(z7,38)(f4,12)]
SHOW
WRITE [(w5,83)(p5,65)(r7,49)(a8,6)(w5,13)]
WRITE [(c4,41)]
WRITE [(s1,43)(k0,43)]
WRITE [(k4,35)]
WRITE [(f8,5)(z4,91)(r5,48)(y8,92)]
READ [o8]
DELETE [f5,z1,a0]
DELETE [a5,w5,f4,m9,i9]
WRITE [(v3,46)]
WRITE [(l6,69)(w7,75)(c0,34)(v8,37)]
WRITE [(x4,11)(o2,73)]
BACKUP
WRITE [(b3,57)(d1,88)(l9,91)(s9,37)]
READ [b8]
DELETE [k7,u0,d9,n9]
WRITE [(e7,30)(w5,16)(k7,10)(i5,79)(k5,53)]
READ [c8]
READ [r0]
WRITE [(f1,24)(b6,76)(z6,72)(q2,59)(e9,89)]
READ [v3]
WRITE [(p1,19)(s5,32)(p2,88)(i1,80)]
WRITE [(c7,41)(m6,29)]
DELETE [f8]
BACKUP
READ [o2]
WRITE [(s2,72)]
DELETE [s2,p7,w6,o8,w1]
WRITE [(b1,59)(h2,47)(q6,65)(y6,61)]
WRITE [(d8,96)(j5,75)(d4,12)(u2,65)(y1,31)]
READ [k0]
DELETE [i0,b2,a2]
WRITE [(y3,87)(j0,14)(p0,6)]
READ [m6]
WRITE [(i9,100)(m2,8)(h4,63)]
READ [j0]
WRITE [(h3,10)(v0,75)(i8,44)(t0,74)]
READ [q2]
WRITE [(k1,21)(n1,64)(b5,59)]
READ [i8]
DELETE [r7,d8,o2]
WRITE [(d3,8)]
READ [r5]
READ [j6]
DELETE [p3,s8,y9,o3]
WRITE [(r3,14)(v6,35)]
DELETE [p2,l6,s9]
DELETE [b1,j4,q6,t3,b5]
READ [l9]
READ [e9]
WRITE [(x8,41)(d5,12)(g7,53)(a2,89)]
WRITE [(m7,15)(a4,5)(m4,77)(l8,66)(l3,25)]
WRITE [(n7,38)(e4,84)(f4,66)(u5,66)(w3,23)]
WRITE [(s2,78)(k3,41)(g2,89)]
WRITE [(e1,12)]
READ [s5]
SHOW
