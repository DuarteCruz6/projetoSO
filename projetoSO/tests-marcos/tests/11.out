[(j4,54)]
[(n2,59)]
(a5, 66)
(b0, 2)
(b8, 49)
(b2, 99)
(c8, 75)
(d2, 46)
(d9, 44)
(d7, 15)
(f5, 32)
(h1, 93)
(i9, 2)
(j6, 16)
(j4, 54)
(k7, 66)
(l2, 77)
(n2, 59)
(p7, 50)
(q7, 32)
(t2, 57)
(w1, 88)
(y9, 2)
[(b2,99)]
[(w6,89)]
(a0, 94)
(a2, 52)
(a5, 66)
(b0, 2)
(b8, 49)
(b2, 99)
(c8, 75)
(d6, 23)
(d2, 46)
(d9, 44)
(d7, 15)
(f4, 12)
(f5, 32)
(h1, 93)
(i0, 64)
(i9, 2)
(j6, 16)
(j4, 54)
(k7, 66)
(l2, 77)
(m9, 6)
(n9, 20)
(n2, 59)
(o8, 1)
(o3, 16)
(p3, 19)
(p7, 50)
(q7, 32)
(r0, 34)
(s8, 28)
(t3, 63)
(t2, 57)
(u0, 82)
(w6, 89)
(w1, 88)
(y9, 2)
(z7, 38)
(z1, 42)
[(o8,1)]
[(b8,49)]
[(c8,75)]
[(r0,34)]
[(v3,46)]
[(o2,73)]
[(k0,43)]
[(m6,29)]
[(j0,14)]
[(q2,59)]
[(i8,44)]
[(r5,48)]
[(j6,16)]
[(l9,91)]
[(e9,89)]
[(s5,32)]
(a4, 5)
(a2, 89)
(a8, 6)
(b6, 76)
(b3, 57)
(b0, 2)
(b8, 49)
(c7, 41)
(c0, 34)
(c4, 41)
(c8, 75)
(d5, 12)
(d3, 8)
(d4, 12)
(d1, 88)
(d6, 23)
(d2, 46)
(d7, 15)
(e1, 12)
(e4, 84)
(e9, 89)
(e7, 30)
(f4, 66)
(f1, 24)
(g2, 89)
(g7, 53)
(h3, 10)
(h4, 63)
(h2, 47)
(h1, 93)
(i8, 44)
(i9, 100)
(i1, 80)
(i5, 79)
(j0, 14)
(j5, 75)
(j6, 16)
(k3, 41)
(k1, 21)
(k5, 53)
(k7, 10)
(k4, 35)
(k0, 43)
(l3, 25)
(l8, 66)
(l9, 91)
(l2, 77)
(m4, 77)
(m7, 15)
(m2, 8)
(m6, 29)
(n7, 38)
(n1, 64)
(n2, 59)
(p0, 6)
(p1, 19)
(p5, 65)
(q2, 59)
(q7, 32)
(r3, 14)
(r5, 48)
(r0, 34)
(s2, 78)
(s5, 32)
(s1, 43)
(t0, 74)
(t2, 57)
(u5, 66)
(u2, 65)
(v6, 35)
(v0, 75)
(v8, 37)
(v3, 46)
(w3, 23)
(w5, 16)
(w7, 75)
(x8, 41)
(x4, 11)
(y3, 87)
(y1, 31)
(y6, 61)
(y8, 92)
(z6, 72)
(z4, 91)
(z7, 38)
